doc/source/tutorial/stats/sampling.rst

```diff
@@ -0,0 +1,275 @@
+          accept and return floats.
+
+.. note:: One can also pass the SciPy distributions as arguments but it can
+          be slow due to validations and expensive NumPy operations.
+          Moreover, it doesn't always have all the information required
+          by some generators like derivative of PDF for TDR method. Also,
+          most of the distributions in SciPy provide a ``rvs`` method which
+          can be used instead.
+
+In the above example, we have set up an object of the
+:class:`~TransformedDensityRejection` method to sample from a
+standard normal distribution. Now, we can start sampling from our
+distribution by calling the ``rvs`` method:
+
+    >>> rng.rvs()
+    -1.526829048388144
+    >>> rng.rvs((5, 3))
+    array([[ 2.06206883,  0.15205036,  1.11587367],
+           [-0.30775562,  0.29879802, -0.61858268],
+           [-1.01049115,  0.78853694, -0.23060766],
+           [-0.60954752,  0.29071797, -0.57167182],
+           [ 0.9331694 , -0.95605208,  1.72195199]])
+
+.. plot::
+
+    >>> import matplotlib.pyplot as plt
+    >>> fig, ax = plt.subplots()
+    >>> x = np.linspace(norm.ppf(0.01), norm.ppf(0.99), 100)
+    >>> # PDF
+    >>> pdf = norm.pdf(x)
+    >>> ax.plot(x, pdf, "-", lw=5, alpha=0.6)
+    >>> # Empirical PDF
+    >>> sample = rng.rvs(size=1000)
+    >>> # discrete samples
+    >>> delta = np.max(pdf) * 5e-2
+    >>> ax.plot(sample[:100], -delta - delta * np.random.random(100), "+k")
+    >>> ax.set_title("PDF")
+    >>> ax.set_ylabel(r"$f$")
+    >>> ax.set_xlabel(r"$x$")
+    >>> plt.show()
+
+.. note:: Please note the difference between the `rvs` method of the
+          distributions present in :mod:`scipy.stats` and the one provided
+          by these generators. Even if the same URNG (``seed``) is used,
+          the resulting ``rvs`` will be different in general: the implementation
+          of `rvs` in :class:`~rv_continuous` usually relies on the NumPy
+          module `np.random` for well-known distributions (e.g., for the normal
+          distribution, the beta distribution) and transformations of other
+          distributions (e.g., normal inverse Gaussian `norminvgauss` and the
+          lognormal distribution). If no specific method is implemented,
+          `rv_continuous` defaults to a numerical inversion method of the CDF
+          that is very slow. As UNU.RAN transforms uniform random numbers
+          differently than SciPy or NumPy, the resulting stream of RVs is
+          different even for the same stream of uniform random numbers. For
+          example, the random number stream of SciPy's ``~norm`` and UNU.RAN's
+          :class:`~TransformedDensityRejection` would not be the same even for
+          the same seed/random_state:
+
+          >>> from scipy.stats import norm, TransformedDensityRejection
+          >>> 
+          >>> dist = StandardNormal()
+          >>> rng = TransformedDensityRejection(dist, seed=123)
```

Comment:
Fixed seed

Reply:
Oops, forgot this one.

Reply:
This is to demonstrate that even for the same seed, RVS of numpy distributions and UNU.RAN can differ.

Reply:
Ok but then I would use something less digest like you have wit `0x7344aca86f1dafd8820be998b639551c`

Reply:
I would be fine with truncating a bit this value like so `0x7344aca86f1` for instance. But definitely not 123 or something like this please. You will always have people copy pasting code and changing the bare minimum...

Reply:
I have removed the explicit seed and just made a copy. Does it look good now?

Reply:
I personally prefer this approach, thanks.